### PR TITLE
Change table name with its values changes + show CRUD permissions

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

**Comment by Coquinho** [comment]
This uses #8

File: src/main/webapp/js/grapheditor/Shapes.js [hunk 1]
```diff
@@ -3621,11 +3621,68 @@ function getShape(cell) {
...
+    if(style.dpRead === 1) {
+      permission += "R";
+    }
+    if(style.dpUpdate === 1) {
+      permission += "U";
+    }
+    if(style.dpDelete === 1) {
+      permission += "D";
+    }
+    permission += ")";
```
Comment (JPTIZ) [inline src/main/webapp/js/grapheditor/Shapes.js:3651]: 1. Typo (Contol -> Control)
2. I would make it a simple function:

```suggestion
    const makePermissionString(create, read, update, delete) {
      return [
          create === 1 ? 'C' : '',
          read === 1 ? 'R' : '',
          update === 1 ? 'U' : '',
          delete === 1 ? 'D' : '',
      ].join('');
    }
    permission = " (Control: " + makePermissionString(
      style.cpiCreate,
      style.cpiRead,
      style.cpiUpdate,
      style.cpiDelete,
    );
    permission += " Dpath: " + makePermissionString(
      style.dpCreate,
      style.dpRead,
      style.dpUpdate,
      style.dpDelete,
    );
    permission += ")";
```

File: src/main/webapp/js/grapheditor/Shapes.js [hunk 2]
```diff
@@ -3621,11 +3621,68 @@ function getShape(cell) {
...
+    console.log("permission", permission)
+    return permission;
   }
 
+  override(mxGraphModel, 'valueForCellChanged', function(cell, value) {
+    var graph = globalGraph;
```
Comment (JPTIZ) [inline src/main/webapp/js/grapheditor/Shapes.js:3657]:
```suggestion
    const graph = globalGraph;
```

**Review by JPTIZ — APPROVED** [review]
The user still have to rename the table in order to CRUD permissions start to update on the title, but...it is enough for now.

**Pull request merged**